.github/workflows/validation.yml

```diff
@@ -0,0 +1,32 @@
+name: hear-validator on baselines
```

Comment:
Remove the other file then? https://github.com/neuralaudio/hear-validator/blob/validation/.github/workflows/test-validation.yml

Again, would still be good to test some of these with loading weight files